<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlphaStim</class>
 <widget class="QMainWindow" name="AlphaStim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlphaStim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="power_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::checked{
	background:rgb(237, 212, 0)
}</string>
    </property>
    <property name="text">
     <string>Power On/Off</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="screen">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>411</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(238, 238, 236)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QProgressBar" name="battery">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>374</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	border: 2px solid grey;
	border-radius: 5px;
	text-align: center;
}
QProgressBar::chunk:enabled {
	background-color:rgb(138, 226, 52);
	border-radius: 5px;
}
QProgressBar::chunk:!enabled {
	background-color:rgb(238, 238, 236);
	border-radius: 5px;
}</string>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>Battery %p%</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="timer">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>374</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>50</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_display">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>374</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextEdit::disabled{
color:rgb(0, 0, 0);
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="intensity_level">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>374</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {
	border: 2px solid grey;
	border-radius: 5px;
	text-align: center;
}
QProgressBar::chunk {
	background-color:rgb(138, 226, 52);
	border-radius: 5px;
	width: 35px;
	margin: 0.5px
}</string>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string>Intensity Level %vÎ¼a</string>
     </property>
    </widget>
    <widget class="QLabel" name="intensity_1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>220</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="intensity_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="intensity_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>220</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="intensity_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>220</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="intensity_5">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Test Circuit:</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_circuit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(239, 41, 41)</string>
     </property>
     <property name="text">
      <string>Disconnected</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="controls">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(186, 189, 182)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="yes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="no">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>No</string>
     </property>
    </widget>
    <widget class="QPushButton" name="show_records">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Show/Hide Records</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPushButton" name="start">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="frequency_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Frequency (Hz)</string>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_label_0">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>100</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="waveform_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform</string>
     </property>
    </widget>
    <widget class="QLabel" name="waveform_label_0">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Alpha</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="waveform_label_1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Beta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="duration_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Timer (min)</string>
     </property>
    </widget>
    <widget class="QLabel" name="duration_label_0">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>20</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="duration_label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>180</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>60</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="duration_label_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>40</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="waveform_label_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Gamma</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="frequency_label_1">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>77</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>16</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="electrode_1">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>34</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::checked{
	background:rgb(237, 212, 0)
}</string>
    </property>
    <property name="text">
     <string>Electrode 1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="electrode_2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>144</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton::checked{
	background:rgb(237, 212, 0)
}</string>
    </property>
    <property name="text">
     <string>Electrode 2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QFrame" name="admin">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>240</y>
      <width>531</width>
      <height>421</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="test_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="swap_batteries">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Swap Batteries?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Test 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>120</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggles power on and off several times.</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_title">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tests should be run from POWERED OFF+DISCONNECTED state and separately.</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connects and disconnects electrodes to show test circuit.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Increases and decreases the current intensity checking min-max bounds.</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>210</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Configures and starts a therapy session.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 4</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Forces a low battery state and checks for warnings.</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>360</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulates a fault which causes 700Î¼a intensity.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Automated Tests:</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>240</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Check basic system configuration</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 5</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Therapy Session</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 6</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>390</y>
       <width>271</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Read console logs for state verification steps.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="test_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>81</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Test 7</string>
     </property>
    </widget>
    <widget class="QLabel" name="test_label_7">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>411</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Simulate 30 minute auto off</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Admin Controls</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
